templates/includes/sidebar.html

```diff
@@ -1,4 +1,10 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <a href="{% url "taxi:driver-detail" pk=user.id %}">{{ user.username }}</a>
```

Comment:
Ensure that the `driver-detail` URL pattern is correctly defined in your `urls.py` with the `pk` parameter, as this link relies on it to navigate to the user's detail page.